pom.xml

```diff
@@ -861,9 +858,7 @@
                <configuration>
                   <forkCount>1</forkCount>
                   <reuseForks>true</reuseForks>
-                  <testFailureIgnore>${testFailureIgnore}</testFailureIgnore>
                   <runOrder>alphabetical</runOrder>
-                  <redirectTestOutputToFile>${maven.test.redirectTestOutputToFile}</redirectTestOutputToFile>
```

Comment:
Not strictly related, but since both are using the standard surefire properties it is superfluous, so when removing the other configuration element I took this out too.